### PR TITLE
templates: allow using matomo analytics

### PR DESCRIPTION
**Pull request by vmiklos**
Similar to Google Analytics, but something you can self-host.

The only difference is that you need to specify an URL as well, since you typically self-host Matomo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ Supports a number of common global variables but patches are welcomed if you nee
 
 - `GOOGLE_ANALYTICS` to use Google Analytics, set this var to your UA-XYZ code
 
+- `MATOMO_URL` to use Matomo, set this var to your server, e.g. "analytics.example.com"
```
Comment (gfidente) [inline README.md:34]: would it make sense to say http://analytics.example.com here, that is, including the http:// prefix?

File: README.md [hunk 2]
```diff
@@ -31,6 +31,10 @@ Supports a number of common global variables but patches are welcomed if you nee
...
+
+- `MATOMO_SIDE_ID` to use Matomo, set this var to your side ID, e.g. "1"
```
Comment (gfidente) [inline README.md:36]: I think it's SITE_ID

File: README.md (hunk 1 above)
Comment (vmiklos) [reply README.md:34]: Actually http or https will be automatically prepended, given that the template uses "//" before the string. But you're right, it makes sense to make this explicit in the README, done now.

File: README.md (hunk 2 above)
Comment (vmiklos) [reply README.md:36]: Yes, all the site -> side was just a typo, fixed now.

**Comment by vmiklos** [comment]
Timeout after a week, let's get this in.

**Pull request merged**